City-of-Helsinki/infofinland-ui
# Misc/messages swr

- update to latest next-drupal
- use translatePath instead of custom resolvePath
- ditch index.jsx, use [[...slug]]
- use type and field_layout to define page component
- use field_layout to decide which menus should be fetched
- remove messages, municipalities and feedback page from prebuild. load them in the client when rendered using swr
